src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,152 @@
+# dotnet/dotnet - Home of the .NET VMR (experimental)
+
+- [Limitations](#limitations)
+- [Goals](#goals)
+- [Building the VMR](#building-the-vmr)
+- [List of components](#list-of-components)
+- [Filing Issues](#filing-issues)
+- [Useful Links](#useful-links)
+- [.NET Foundation](#net-foundation)
+- [License](#license)
+
+This repository is an **Experimental Virtual Monolithic Repository** that hosts all the code to help you build the entire .NET product end-to-end from source (often referred to as source-build).
+
+What this means:
+- **Monolithic** - a join of multiple individual repositories that make up the whole product, such as [dotnet/runtime](https://github.com/dotnet/runtime) or [dotnet/sdk](https://github.com/dotnet/sdk).
+- **Virtual** - not a replacement for the individual repositories that the product consists of but rather a mirror where sources of the individual repositories are synchronized into.
+- **Experimental** - not to be depended on as we reserve the right to delete the current instance and create a new, different one in its stead.
```

Comment:
I think we should provide/link to a road map.  I think it is important to convey the commitment to this approach and when "dependencies" can be made on it.

Reply:
Do you mean this https://github.com/mmitche/arcade/blob/ub-docs/Documentation/UnifiedBuild/README.md?

Or do you mean a VMR-only more granular roadmap?

Reply:
VMR granular.  In context of the description here, I think we should be including a timeframe in which we expect the "experimental" nature to be removed do that folks "can take a dependency" on this.

Reply:
How about "by .NET 8 Preview 1". I hope it is before that, but that feels like a good outside bound to commit to now.

Reply:
I had a note to the `Limitations` section (plus a reference up here) with a link to the detailed roadmap that will be in the UB docs.

I tried to put a roadmap here as well but it was just getting very long and if I omitted parts, the rest didn't make a lot of sense so I think it's better to have the full thing in the UB?

Reply:
This has been resolved in the other thread. We now make it clear you can expect .NET 8 Linux builds from the VMR